packages/fxa-auth-server/lib/oauth/jwt_access_token.js

```diff
@@ -43,6 +48,14 @@ exports.create = async function generateJWTAccessToken(accessToken, grant) {
     claims['fxa-subscriptions'] = grant['fxa-subscriptions'].join(' ');
   }
 
+  if (grant.generation) {
+    claims['fxa-generation'] = grant.generation;
+  }
+
+  if (grant.profileChangedAt) {
+    claims['fxa-profileChangedAt'] = grant.profileChangedAt;
+  }
```

Comment:
I don't object to it, but I'm surprised anyone would want/need to get `profileChangedAt` our of here, is this in service of a specific consumer?

Reply:
Oh, I see, I guess this is so you can return it from `/verify` endpoint? 👍

Reply:
yes, though I don't know how it's used downstream of /verify. I'm blindly adding it because it exists in the other case.

Reply:
I expect it's used by the profile-server to help with cache invalidation in some way. Can't guarantee that other RS's aren't also using it, but I'd be surprised.

Reply:
> I expect it's used by the profile-server to help with cache invalidation in some way

Yea that is how it is currently used by the profile-server. It never gets returned to [reliers](https://github.com/mozilla/fxa/blob/4ab4c64e3a1a847291d0dd8d130f9dc698283e80/packages/fxa-profile-server/lib/routes/profile.js#L54) from profile server.